jenkins-x.yml

```diff
@@ -121,17 +121,17 @@ pipelineConfig:
             - tmp-website
             - --enableGitInfo
             - --baseURL http://${APP_NAME}.jx-${REPO_OWNER}-${REPO_NAME}-pr-${PULL_NUMBER}.${DOMAIN}/
-          - image: chabad360/htmlproofer:latest
-            name: htmlproofer
-            command: htmlproofer
-            args:
-              - ./tmp-website
-              - --allow-hash-href
-              - --disable-external
-              - --check-html
-              - --checks-to-ignore ScriptCheck
-              - --empty-alt-ignore
-              - --file-ignore /.*apidocs.*/,/.*\/zh.*/,/.*\/es.*/,/.*\/docs\/reference\/api.*/,/.*\/docs\/reference\/config.*/,/.*\/community\/events.*/
+          # - image: chabad360/htmlproofer:latest
```

Comment:
I guess all this moving around things has broken the HTML checker. I guess you are working on sorting this out, right?

Reply:
The HTML checker is disabled temporarily yes until the structure is finalized.  For now, as you and I spoke, users will go to the major areas via the left navigation.